graphbolt/src/csc_sampling_graph.cc

```diff
@@ -268,8 +269,97 @@ inline torch::Tensor UniformPick(
     picked_neighbors =
         torch::randint(offset, offset + num_neighbors, {fanout}, options);
   } else {
-    picked_neighbors = torch::randperm(num_neighbors, options);
-    picked_neighbors = picked_neighbors.slice(0, 0, fanout) + offset;
+    if (replace) {
+      picked_neighbors =
+          torch::randint(offset, offset + num_neighbors, {fanout}, options);
+    } else {
```

Comment:
The conflict was poorly addressed, can you double check here?

Reply:
done